### DISCUSSION TITLE
New version with themes, do I need to update screen for it or just update motherboard?

### DISCUSSION THREAD

**Discussion by VashBaldeus** [Jyers/Marlin#551] [Q&A] [answered]
New version with themes, do I need to update screen for it or just update motherboard?

**Comment by Tuncay-Ayhan** [comment, marked as answer]
> New version with themes, do I need to update screen for it or just update motherboard?


You only need to flash the firmware on your motherboard. If you want custom icons, you can flash the display to. It's not necessary though.
